helm-chart/values.yaml.changelog.md

```diff
@@ -7,6 +7,10 @@ Please follow this convention when adding a new row
 
 ----
 
+## Upgrading to Renku 0.7.10
```

Comment:
Thanks for adding this line already. 0.7.10 is already past, os I guess we should just add **unreleased** as title here and modify it when we do a release.